docs/reference/index.md

```diff
@@ -0,0 +1,30 @@
+The Pulsar / Atom Backend happily accepts all form of contributions, and contributions are always welcome.
```

Comment:
```suggestion
# Reference Index

The Pulsar / Atom Backend happily accepts all form of contributions, and contributions are always welcome.
```
Not sure about the wording I've suggested for the title but can't think of something better...